iphone/Classes/MediaModule.m

```diff
@@ -2016,7 +2016,7 @@ - (void)popoverPresentationControllerDidDismissPopover:(UIPopoverPresentationCon
 - (void)presentationControllerDidDismiss:(UIPresentationController *)presentationController
 {
 #if defined(USE_TI_MEDIASHOWCAMERA) || defined(USE_TI_MEDIAOPENPHOTOGALLERY) || defined(USE_TI_MEDIASTARTVIDEOEDITING)
-#if IS_SDK_IOS_14
+#if IS_SDK_IOS_14 && defined(USE_TI_MEDIAOPENPHOTOGALLERY)
```

Comment:
Isn't this covered by the line above?

Reply:
Yes, but it also goes into the statement if I only use `Ti.Media.showCamera` or `Ti.Media.startVideoEditing`. So it throws the compile error once we only the above, but not `Ti.Media.openPhotoGallery`

Reply:
Ah, gotcha. If one of the other OR segments passes, we don't guard properly. OK!